lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -409,6 +409,160 @@ def test_kwargs(self):
         self.assertMaskedArrayEqual(expected_result, res_cube.data)
 
 
+class Testslices(tests.IrisTest):
+    '''
+    This class tests the slices capability of iris.cube.Cube
+    '''
+    def setUp(self):
+        '''
+        setup a 4D iris cube, each dimension is length 1.
+        The dimensions are;
+            dim1: height
+            dim2: time
+            dim3: longitude
+            dim4: latitude
```

Comment:
@djkirkham I am tempted to not use stock cubes for this setup as I don't really see the need, and if I did I would have to change all of the tests that have been written here.  I am also not sure that it is necessary to change round the order of dimensions; although they are not proper here, if I changed them around then I would also have to change all of the tests.  And how much does it matter, really?